Stream/CloudDataFlowLogSink/Dockerfile

```
@@ -1,11 +1,11 @@
 #See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
-FROM mcr.microsoft.com/dotnet/aspnet:5.0-buster-slim AS base
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
 WORKDIR /app
 EXPOSE 80
 EXPOSE 443
 
 FROM base AS final
 WORKDIR /app
-COPY bin/Debug/netcoreapp3.1/publish .
+COPY bin/Debug/net8.0/publish .
```

```suggestion
COPY bin/Debug/net6.0/publish .
```

Is Debug still correct? I noticed the last dockerfile I looked at used Release